config.schema.yaml

```diff
@@ -4,3 +4,10 @@ api_key:
   type: "string"
   secret: true
   required: true
+region:
+  description: "The service region for your OpsGenie account"
+  type: "string"
+  enum:
```

Comment:
Making this an `enum` creates a maintenance burden, since now we will need to update the enum every time OpsGenie adds a region. I would remove the `enum` and add `required: false`.

Reply:
I've addressed this, the `enum` was attempting to stop someone adding something invalid and wondering why it did not work. I expect adding a region would be rare.